### PR TITLE
Add a new process: pressed

### PR DESCRIPTION
**Pull request by aleene**
**Description:**
process pressed and variants

### CODE REVIEW

File: taxonomies/ingredients_processing.txt [hunk 1]
```diff
@@ -372,6 +372,45 @@ fr:extrait naturel de, extrait naturel d'
...
+# en:false:compressed
+de:abgepresste
+es:espremido de
+fr:pressé, pressée, pressées
+#fr:false:à pâte pressée, fromage pressé
+it:spremuto di
+sv:presset
+
+#<en:pressed
+en:pressed with alkali
```
Comment (stephanegigandet) [inline taxonomies/ingredients_processing.txt:385]: that's weird, shouldn't it be "processed with alkali"

Comment (aleene) [reply taxonomies/ingredients_processing.txt:385]: I don't know, thats what I saw in the ingredients.

Comment (stephanegigandet) [reply taxonomies/ingredients_processing.txt:385]: probably a mistake in the ingredients, let's not add it (unless we do find several products that contain it on the ingredients photo)

Comment (teolemon) [inline taxonomies/ingredients_processing.txt:385]:
```suggestion
```

**Pull request merged**